code/style.css

```diff
@@ -1,3 +1,335 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  margin: 10px;
+  background-color: black;
+  font-family: 'Itim', cursive;
+  color: aliceblue;
+}
+
+.background-image {
+  position: fixed;
+  top: 0;
+  left: 0;
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+  opacity: 0.4;
+  z-index: -1; /* keep the image behind the content */
+}
+
+.octacat {
+  width: 18px;
+  height: 18px;
+  background-color: white;
+  border-radius: 50%;
+  padding: 2px;
+  margin: 0 6px 0 10px;
+}
+
+.header-title {
+  position: relative;
+  font-size: 18px;
+  display: flex;
+}
+
+.profile {
+  width: 90%;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.profile-info {
+  grid-column: span 1;
+  font-size: 12px;
+  width: 100%;
+  height: 100%;
+}
+
+table {
+  width: 100%;
+}
+
+td {
+  width: 50%;
+}
+
+.profile-image-icons {
+  grid-column: span 1;
+  width: 90%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.profile-image {
+  border: 1px solid #ccc;
+  border-radius: 50%;
+  width: 74px;
+  height: 74px;
+  margin-bottom: 10px;
+  filter: drop-shadow(8px 8px 16px #ccc);
+  transform: scale(1);
+  transition: 0.3s ease-in-out;
+}
+
+.profile-image:hover {
```

Comment:
The hover effect looks nice! But it also makes me expect that it should link to something...

Reply:
Yeah, you are right, not so user friendly. I just wanted to try out different effect 😄